branchff

```diff
@@ -175,8 +175,7 @@ logrun -s git merge -X ours $MASTER_OBJECT
 # <snip>
 # Commit your changes in another terminal and then continue here by pressing enter.
 
-USEFUL_UPDATES=(update-openapi-spec generate-docs 
-                update-federation-openapi-spec)
+USEFUL_UPDATES=(update-openapi-spec update-federation-openapi-spec)
```

Comment:
Should we also run `update-generated-docs` now to ensure the stubs are all right?

Reply:
The branch will no longer ever be "polluted" with generated docs and further, when branchff has fast-forwarded it to master, that in and of itself will ensure the stubs are there.  There's no need to run update-generated-docs.sh at this point.  If master somehow gets trashed, verify-generated-docs.sh will catch it long before branchff is ever run.

Reply:
This should go in before today's branchff.